x/crosschain/keeper/evm_hooks_test.go

```diff
@@ -714,27 +715,6 @@ func TestKeeper_ProcessLogs(t *testing.T) {
 		require.Len(t, cctxList, 0)
 	})
 
-	t.Run("error returned for invalid event data", func(t *testing.T) {
```

Comment:
Can't we keep this test using a invalid address?

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/2bcdfbc727c12612159ffbf008bb00505a996b4b